src/popup/components/AddressTruncated.vue

```diff
@@ -11,7 +11,7 @@
       :protocol="protocol"
       class="address-truncated-protocol"
     />
-    <div class="address-truncated-chunks">
+    <div v-if="!hideAddress" class="address-truncated-chunks">
```

Comment:
If the address is hidden this component becomes just a ProtocolIcon which sounds rather weird. If you want to replace the address with something different you can use slot to wrap the contents of this div.

```suggestion
    <div v-if="!hideAddress" class="address-truncated-chunks">
```